continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+python:
+    - 3.7
+    - 3.8
+    - 3.9
+    - 3.10
```

Comment:
Should we be building 3.10 even though we haven't added it to CI here?

Reply:
Probably not; I initially added it because conda-forge is publishing 3.10 builds, but in general I can't think of too many specific use cases where we would need nightly 3.10 builds right now